solr/core/src/java/org/apache/solr/security/JWTPrincipal.java

```diff
@@ -77,10 +75,15 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "JWTPrincipal{" +
-        "username='" + username + '\'' +
-        ", token='" + "*****" + '\'' +
-        ", claims=" + claims +
-        '}';
+    return "JWTPrincipal{"
+        + "username='"
+        + username
+        + '\''
```

Comment:
this has become more awkward. It could be improved by combining the apostrophe into the string of the next line; same for further below.

Reply:
Used String.format() instead